mods/d2k/rules/vehicles.yaml

```diff
@@ -337,6 +337,11 @@ devastator:
 		Step: 5
 		Ticks: 3
 		HealIfBelow: 50%
+	AttackDetonates:
```

Comment:
For the MAD tank it is both a deploy and an attack order. For a unit with another `Attack*` trait like the Devastator the `AttackFrontal` takes precedence.